students/rhamersky/Lesson_10/mailroom_part6.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python3
+# Description: This program will allow a user to input a donor and a donation.
+# The user will also be able to send out thank you notes to the donors.
+# Developer: Ryan Hamersky
+# Date: 05/21/2018
+# Rev: New
+
+# -----Data Section-----
+str_name = str()    # --> User's name
+int_user_input = int()  # --> User's option.
+dic_donors_sorted = {}  # --> Sorted dictionary of donors.
+dic_donors = {"William Gates, III": [10000.0, 1], "Mark Zuckerberg": [20000.0, 1],
```

Comment:
If you're going to indicate type in your variable names, I would use the actual type name rather than abbreviating further (dict to dic) for readability